src/components/atomic/Checkbox.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { InputHTMLAttributes } from "react";
+import classNames from "classnames";
+
+type CheckboxProps = InputHTMLAttributes<HTMLInputElement> & {
+  labelText?: string;
+};
+
+const Checkbox = ({ className = "", ...props }: CheckboxProps) => {
+  const wrapperStyles = classNames({
+    ["h-5 w-5 relative"]: true,
+  });
+  const checkboxStyles = classNames({
+    ["h-5 w-5 border-2 border-primary rounded appearance-none \
+      checked:bg-primary checked:fill-white -z-1 absolute"]:
+      true,
+  });
+  const svgStyles = classNames({
+    ["h-5 w-5 fill-none stroke-2 absolute"]: true,
+  });
+
+  return (
+    <label className={wrapperStyles} htmlFor={props.id}>
+      <svg className={svgStyles}>
+        <path
+          xmlns="http://www.w3.org/2000/svg"
+          d="M 3,13 7,17 17,3"
+          style={{ stroke: "#fff", height: 20, strokeWidth: 2, fill: "none" }}
+        />
+      </svg>
+      <input
+        type="checkbox"
+        {...props}
+        className={checkboxStyles}
+        id={props.id}
+      />
+      <span style={{ marginLeft: "25px" }}>
```

Comment:
Avoid inline styles (use tailwind!!)

Also, ideally we use the Text component here?

Reply:
Also should the checkbox and text be bottom-aligned or middle-aligned?

Reply:
made it middle aligned--hope that's ok